Allow uneven amount of brackets

josojo opened the pull request:
closes #234 

test plan: there is no logic depended on whether we have an even amount of brackets.  Hence, no tests are needed. It's really just a relic of the past.

bh2smith commented:
> test plan: there is no logic depended on whether we have an even amount of brackets. Hence, no tests are needed. It's really just a relic of the past.

By test plan, I meant provide a command I can run to try this.

josojo commented:
>I'd like to see the scripts running on Rinkeby with 1 bracket or 19 just to be sure everything is right.

What would you check then on the rinkeby instances? everything? I added some tests to ganache, it increases the travis run-time, but at least here we can check efficiently want is happening.

> By test plan, I meant to provide a command I can run to try this.

Since there is no single logic related to the evenness of the size, I can not provide a specific test. I just added a bunch of test cases and amend some tests
